src/macaron/config/defaults.ini

```diff
@@ -494,7 +503,7 @@ artifact_extensions =
 # Package registries.
 [package_registry]
 # The allowed time range (in seconds) from a deploy workflow run start time to publish time.
-publish_time_range = 3600
+publish_time_range = 7200
```

Comment:
I wonder why did we decide to increase the publish time range ?

Reply:
Based on the artifacts I have experimented with, `publish_time_range` might need larger values, but this value can be adjusted by users as required.